client/me/profile-links/index.jsx

```diff
@@ -173,20 +172,21 @@ class ProfileLinks extends React.Component {
 
 	render() {
 		return (
-			<div>
+			<Fragment>
 				<QueryProfileLinks />
-				<SectionHeader label={ this.props.translate( 'Profile Links' ) }>
-					<AddProfileLinksButtons
-						showingForm={ this.state.showingForm }
-						onShowAddOther={ this.showAddOther }
-						showPopoverMenu={ this.state.showPopoverMenu }
-						onShowAddWordPress={ this.showAddWordPress }
-						onShowPopoverMenu={ this.showPopoverMenu }
-						onClosePopoverMenu={ this.closePopoverMenu }
-					/>
-				</SectionHeader>
-				<Card>{ this.state.showingForm ? this.renderForm() : this.renderProfileLinks() }</Card>
-			</div>
+					<SectionHeader label={ this.props.translate( 'Profile Links' ) }>
+						<AddProfileLinksButtons
```

Comment:
Looks like some extraneous indentation snuck in here. Have you been able to get the git hooks setup @torres126? They should auto-format on save.

Reply:
Hmm, nope, I'm not aware what that is. I'll look into that when I get the chance. :) 

(Before a few days ago, my PRs were made on my tablet which kept erasing lines and I couldn't use tab, so I think many of my PRs may have some odd spacing as a result too)

Reply:
I'm not sure why they aren't working as they should be Automattic. The hooks are in `.git/hooks`

Can you verify that they are executable?
Are you on Windows?
Are you committing via the `git` command line utility or using some GUI?

Reply:
> Can you verify that they are executable?

Me isn't sure how to do that :sweat_smile: 

> Are you on Windows?

Android for all my previous PRs until a few days ago, where I'm on Mac

> Are you committing via the git command line utility or using some GUI?

Directly on github.com

Reply:
That makes sense then if you've been editing code through the browser. As you get into a workflow on macOS the auto-formatting should run on `git commit`